### PR TITLE
Update CPC meeting doc template

### PR DESCRIPTION
**Pull request by tobie**
/cc @joesepi

### CODE REVIEW

File: templates/minutes_base_cross_project_council [hunk 1]
```diff
@@ -16,20 +16,36 @@ $OBSERVERS$
...
+### Next week's working session
 
 Are there any initiatives or agenda items that we should use a working session to further progress on?
+- https://github.com/openjs-foundation/cross-project-council/labels/cpc-working-session
+
+###  
+
+Are there any issues we would need to review?
+
+### Regular reviews
```
Comment (tobie) [inline templates/minutes_base_cross_project_council:40]:
```suggestion
```

File: templates/minutes_base_cross_project_council [hunk 2]
```diff
@@ -16,20 +16,31 @@ $OBSERVERS$
...
 
-## Next week's working session
+### Next week's working session
 
 Are there any initiatives or agenda items that we should use a working session to further progress on?
+- https://github.com/openjs-foundation/cross-project-council/labels/cpc-working-session
+
 
-## Dates and Reminders
+Please review regularly our list of dates and reminders, our quarterly review issues, and check the list of issues that can be closed:
```
Comment (tobie) [inline templates/minutes_base_cross_project_council:37]:
```suggestion
### Regular reviews

Please review regularly our list of dates and reminders, our quarterly review issues, and check the list of issues that can be closed:
```

**Review by joesepi — APPROVED** [review]
LGTM. Thanks!

**Comment by tobie** [comment]
@mhdawson could you please review and merge this? Thank you!

**Review by mhdawson — APPROVED** [review]
LGTM

**Pull request merged**